admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -219,7 +295,7 @@ private void setStatusRelatedFields(ReplicationField field, ReplicatorConfig con
     field.itemsTransferred((int) itemsTransferred);
   }
 
-  private @Nullable ReplicationSiteField getSiteFieldForSite(ReplicationSite site) {
+  public @Nullable ReplicationSiteField getSiteFieldForSite(ReplicationSite site) {
```

Comment:
❓ javadoc since it is now public
✏️ `site` should be annotated with `@Nullable`